### DISCUSSION TITLE
显示版本过低 无法登陆

### DISCUSSION THREAD

**Discussion by MunRen** [huan/docker-wechat#115] [Q&A]
扫码登陆现实版本低，而且页面被遮挡会现实一个轮廓
![2021-01-13_16-24](https://user-images.githubusercontent.com/43802357/104425850-1e918d00-55bc-11eb-9c17-01c0b2be3e74.png)

**Comment by TurbochargedSnail** [comment]
显示版本过低，登录不进去，怎么解？
新版本微信是否可加进来？
